main.sh

```diff
@@ -9,8 +9,9 @@ if [ 0 -eq "$(wc -l <"$tempfile")" ]; then
 	exit 0
 fi
 
-if [ "$EVENT_NAME" != "pull_request" ]; then
-	exit 1
+# Check if EVENT_NAME is either pull_request or pull_request_target
+if [ "$EVENT_NAME" != "pull_request" ] && [ "$EVENT_NAME" != "pull_request_target" ]; then
+    { echo "Error: Invalid event name. Expected either 'pull_request' or 'pull_request_target'."; exit 1; }
```

Comment:
```suggestion
    exit 1
```

The error message is unnecessary because the event is valid.
The reason the action fail is not the event is invalid but the code isn't formatted.

Reply:
Hmm. It's good to output error log using `error command`.

https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-error-message

```sh
echo "::error Files aren't formatted"
```